test/Unit/phpunit.xml

```diff
@@ -1,8 +1,8 @@
 <phpunit
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:noNamespaceSchemaLocation="../../vendor/phpunit/phpunit/phpunit.xsd"
+    xsi:noNamespaceSchemaLocation="../../vendor/phpunit/phpunit/schema/9.2.xsd"
```

Comment:
@sanmai 

Thank you for your efforts towards fixing this issue.

I'm not sure whether that's the best path: I would prefer to reference the schema without specifying a version - this way I never need to update it when I update `phpunit/phpunit`.

Perhaps we need to do something upstream, as suggested by Sebastian Bergmann? Provide additional console command options so we can avoid manipulating `phpunit.xml` when running `infection/infection`?

Reply:
We can, but it doesn't seem like to be happening very soon. The question here is whenever you want to use PHPUnit 9.3 with those four new features, or you think you'll be fine with 9.2.